src/components/Amount/Amount.re

```diff
@@ -0,0 +1,68 @@
+[@react.component]
+let make = (~transactions: Types.t) => {
+  let total = ref(0);
+  let income = ref(0);
+  let expense = ref(0);
+
+  let rupee = React.string({js|₹|js});
+
+  let expenseIncomeStyle =
+    ReactDOM.Style.make(
+      ~backgroundColor="white",
+      ~display="flex",
+      ~justifyContent="space-between",
+      ~boxShadow="-2px 3px 3px 0px rgba(168, 168, 168, 0.75)",
+      ~borderRadius="4px",
+      (),
+    );
+  let incomeStyle = ReactDOM.Style.make(~flex="1", ~textAlign="center", ());
+  let incomeH5Style =
+    ReactDOM.Style.make(~color="green", ~fontSize="20px", ());
+  let expenseStyle = ReactDOM.Style.make(~flex="1", ~textAlign="center", ());
+  let expenseH5Style =
+    ReactDOM.Style.make(~color="red", ~fontSize="20px", ());
+
+  let convert = num => {
+    let int_to_string = string_of_int(num);
+    let x = int_to_string |> Js.String.split("-");
+    let string_to_int = int_of_string(x[1]);
+
+    string_to_int;
+  };
+
+  if (transactions != [||]) {
+    for (i in 0 to Array.length(transactions) - 1) {
```

Comment:
It's nice you learned making use of ref. Also, we can make use of `reduce` in array. Like [here](https://rescript-lang.org/docs/manual/latest/api/belt/array#reduce)